### ISSUE TITLE
Accessibility support

### ISSUE THREAD

**Issue by tyrylu** [qTox/qTox#300]
Hello,
it would really help if qtox would support screen reader accessibility. QT accessibility can be done and for testing you can use NVDA on Windows and Orca under Linux.
The current situation is as follows:
- [x] Button labels are read correctly
- [x] The presence of some edit box is announced
- [x] Labels aren't read (see #2101)
- [x] Focus behavior is sometimes quite strange (see #2100)

**Comment by zetok** [comment]
from #1215:

> The QTox Windows Client is not accessible for Users with Assistive Technologies.
> See: http://qt-project.org/doc/qt-4.8/accessible.html

.

> Or consider upgrading to the latest QT5 version, there were some substantial accessibility improvements and the entire qtaccessibblewidgets stuff was simplified.

@tyrylu which Qt version has those improvements?

**Comment by tyrylu** [comment]
Now, when the builds are using QT5, it's better to create smaller issues.

**Comment by suhr** [comment]
Wait. Let this issue be a roadmap of these smaller issues (consider editing issue message).

**Comment by tyrylu** [comment]
Alright, i'll update the issue descripttion with my current findings.

**Comment by ezavod** [comment]
@tyrylu Is there anything left to do in this issue, since #2100 has been closed?
I would say this one could be closed.
